pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -1443,6 +1438,25 @@ public MisoBoxPosition mapRow(ResultSet rs, int rowNum) throws SQLException {
     return assay;
   };
 
+  private static final RowMapper<AssayTest> assayTestRowMapper = (rs, rowNum) -> {
+    AssayTest test = new DefaultAssayTest();
+    test.setName(rs.getString("name"));
+    test.setTissueType(rs.getString("tissueType"));
+    Boolean negate = rs.getBoolean("negateTissueType");
+    if (!rs.wasNull()) {
```

Comment:
It [looks like the column is not null](https://github.com/miso-lims/miso-lims/blob/develop/sqlstore/src/main/resources/db/migration/V8501__assay_tests.sql#L5), is this check still needed?

Reply:
Ah right - thanks for noticing this! The check is wrong as written, but there should be a different check to handle plain sample mode as both of these are specific to detailed sample mode. I'll remove these `wasNull` checks and instead only set the fields if tissue type is not null (tissue type is required in detailed sample mode)

Reply:
Ah, that makes sense!